src/main/java/uk/gov/ch/service/psc/PersonsWithSignificantControlService.java

```diff
@@ -8,5 +8,7 @@
 public interface PersonsWithSignificantControlService {
 
      List<PersonWithSignificantControl> getPersonsWithSignificantControl(String companyNumber, Pageable pageable);
+
+     List<PersonWithSignificantControl> getOverseasEntityData(String companyNumber, Pageable pageable);
```

Comment:
getOverseasEntityPersonsWithSignificantControl()

Reply:
Don't we need a new Model object to pass back? For the JSON fields we want to return?